bosh-setup/azuredeploy.json

```diff
@@ -89,19 +66,42 @@
     },
     "enableDNSOnDevbox": {
       "type": "bool",
+      "defaultValue": true,
       "allowedValues": [
         true,
         false
       ],
       "metadata": {
-        "description": "A default DNS will be setup in the devbox if it is true. Need a manual reboot to take effect."
+        "description": "A default DNS will be setup in the devbox if it is true."
+      }
+    },
+    "tenantID": {
+      "type": "string",
+      "defaultValue": "TENANT-ID",
```

Comment:
Ya please go ahead and edit those parameters again. For storage accounts and dns names, please use the value "#####". The bot will replace this with random gen string. For other parameters any valid value will work.